code/Base/Hud/Controls/ControlGlyph.scss

```diff
@@ -0,0 +1,25 @@
+.controlglyph {
+	align-items: center;
+	.glyph {
```

Comment:
Spacing here and line 12. Also, if we adhere to PascalCase for CSS classes, we should fix that in this class, too.